gravitee-apim-repository/gravitee-apim-repository-jdbc/src/main/java/io/gravitee/repository/jdbc/management/JdbcAuditRepository.java

```diff
@@ -103,6 +104,18 @@ public Optional<Audit> findById(String id) throws TechnicalException {
         }
     }
 
+    @Override
+    public Set<Audit> findAll() throws TechnicalException {
+        LOGGER.debug("JdbcAuditRepository.findAll()");
+        try {
+            List<Audit> items = jdbcTemplate.query(getOrm().getSelectAllSql(), getOrm().getRowMapper());
```

Comment:
Are we sure it will not throw a memory exception for DB with a lot of Audits?
Related to  [`7359868` (#1634)](https://github.com/gravitee-io/gravitee-api-management/pull/1634/commits/73598684ae86b39b57cbed1e2dab4b2faea22495#diff-66cc0adfa6e78c8e36b1bd0a664257d7cc0f7df465e2b1dfdafcececfc94b312L293)

Reply:
Already discuss with @marcambier and he said `pas trop inquiet..`😁  and that it would be nice to test with a large volume of data. 
I haven't done the test yet I would do it in peer programming if there is a voltaire

Reply:
Agree with Gaetan, findAll() is definitely something to avoid in favor of paginated results.